common/journal_entries/imperia_political_discord_journal_entries.txt

```diff
@@ -57,15 +61,21 @@
 		}
 	}
 	current_value = {
-		value = var:political_discord
+		value = root.var:political_discord
```

Comment:
you already should be in root. Or am I missing something?

Reply:
Huh, this was changed when I was playing around with scripted bars I see. (which is not useable for discords purposes, btw)
No idea how it happened.. I'll change it back ;p